content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -552,7 +552,7 @@ npx prisma format
 Environment variables loaded from prisma\.env
 Prisma schema loaded from prisma\schema.prisma
 Error: Schema parsing
-error: The preview feature "orderByRelationnp" is not known. Expected one of: microsoftSqlServer, groupBy, createMany, orderByRelation, nApi
+error: The preview feature "orderByRelation" is not known. Expected one of: microsoftSqlServer, groupBy, createMany, orderByRelation, nApi
```

Comment:
Wasn't showing an error via a wrong preview feature the intent of this example? We probably should replace this with `doesNotExist` or similar.

Although: Why do we document the error case? Nothing is formatted in this case, as the validation fails before formatting starts, right?